presigned-pause.just

```diff
@@ -157,12 +157,12 @@ simulate-all:
       simulate
   done
 
-upload-PSPs:
+upload-PSPs VAULTNAME:
```

Comment:
Nice catch on the missing vault name. Instead of specifying it as an arg, can we infer the vault name automatically based on the directory? e.g. `tasks/eth` uses the mainnet one, `tasks/sep` uses the sepolia one, and other dirs throw an error

Reply:
Oh wanted to something similar actually!
But then realized I do prefer that security oncall choose the vault manually by adding this a param (this will also be consistent with previous doc + new one). 
This will add an small checks extra step. 
Maybe in the future version?

Reply:
> But then realized I do prefer that security oncall choose the vault manually by adding this a param (this will also be consistent with previous doc and new doc created for this).

What is the reason for this preference? It seems less error prone if we can automate it

Reply:
Yes, the main reasons was:
- Be consistent with previous doc we got, keep the usage of the name into the args. 
- Also, personal feeling I prefer with most of the tooling we have having something that clearly specify where we want to work (Mainnet or Sepolia). 
Maybe we can add a small message to mention: 
```
You are currently on a ethereum task Path and you want to submit the PSPs on the Sepolia vault?
```

Reply:
> But then, if there's only 1 allowed value of the arg for a given task, why have the arg in the first place?

Yes you right on this point. I have follow then the first idea and added something that will match automatically from the path if `sep` this will return the sepolia vault and vice versa for mainnet.
Now, no arguments is required.  
I just added a confirmation to make sure this is desired.
 
I took a video for the [doc](https://www.notion.so/oplabs/RB-111-Pre-signed-pause-8bce5b27728b447aa51ba12fcfff9e58?pvs=4#a227c38a09e14518b26095baa91f81c8) and also putting this one here to show you: 

https://github.com/user-attachments/assets/5c05deb7-73bf-49f9-9337-555aa3287a46

Reply:
Nice, thanks a lot for this, the UX looks really great!